tools/cordova/builder.js

```diff
@@ -375,6 +380,23 @@ export class CordovaBuilder {
     });
   }
 
+  configureAndCopyResourceFiles(resourceFiles, iosElement, androidElement) {
+    _.each(resourceFiles, resourceFile => {
+      // copy file in cordova project root directory
+      var filename = resourceFile.src.split('\\').pop().split('/').pop();
```

Comment:
Hmm, is this splitting/popping on `resourceFile.src` in order to account for the different file path separators on Windows vs Unix, or is this because of something else I'm missing?

If it is because of such a reason, can we use Node's [`path.parse`](https://nodejs.org/api/path.html#path_path_parse_path) instead?  A backslash is an uncommon, but possible, character in a valid Unix path.

Reply:
@abernix sorry for this ugly thing, i've updated the PR with path.parse()
regards